.github/workflows/preview-image.yml

```diff
@@ -71,7 +69,12 @@ jobs:
         run: |
           set -x
           VERSION=$(jq -r .version package.json)
-          FULL_VERSION=${VERSION}-b${GITHUB_RUN_ID}.${GITHUB_RUN_NUMBER}
+          BRANCH=${{ event.workflow_run.head_branch }}
+          if [[ "$BRANCH" =~ ^[0-9]+\.[0-9]+\.[0-9]+\-dev ]]; then
+            FULL_VERSION=${VERSION}
+          else
+            FULL_VERSION=${VERSION}-b${GITHUB_RUN_ID}.${GITHUB_RUN_NUMBER}
```

Comment:
Is this what we want? This used to be the short SHA of the commit.

Reply:
it's what it used to be
you can check images, that i've published during testing in a fork
https://hub.docker.com/r/achubatiuk/preview/tags